scalalib/src/mill/javalib/android/AndroidAppModule.scala

```diff
@@ -191,25 +235,23 @@ trait AndroidAppModule extends JavaModule {
    * [[https://developer.android.com/tools/apksigner apksigner Documentation]]
    */
   def androidApk: T[PathRef] = Task {
-    val signedApk: os.Path = T.dest / "app.apk"
+    val signedApk: os.Path = Task.dest / "app.apk"
 
-    os.call(
-      Seq(
-        androidSdkModule().apksignerPath().path.toString,
-        "sign", // Call apksigner tool
-        "--ks",
-        androidKeystore().path.toString, // Path to keystore
-        "--ks-key-alias",
-        "androidkey", // Key alias
-        "--ks-pass",
-        "pass:android", // Keystore password
-        "--key-pass",
-        "pass:android", // Key password
-        "--out",
-        signedApk.toString, // Output signed APK
-        androidAlignedUnsignedApk().path.toString // Use aligned APK
-      )
-    )
+    os.call((
+      androidSdkModule().apksignerPath().path,
+      "sign",
+      "--ks",
+      androidKeystore().path,
+      "--ks-key-alias",
+      "androidkey",
+      "--ks-pass",
+      "pass:android",
+      "--key-pass",
+      "pass:android",
+      "--out",
```

Comment:
this is only for the debug key, obviously. If someone want to build a signed **release** package, it should be an option to provide a singing configuration: release key and credentials for it.

Reply:
No worries we will add a function which will take args from user and then sign...